src/Tizen.WebView/Tizen.WebView/WebView.cs

```diff
@@ -328,6 +328,102 @@ public void SetFocus(bool focused)
             Interop.ChromiumEwk.ewk_view_focus_set(_realHandle, focused);
         }
 
+        /// <summary>
+        /// Get coordinates of content.
+        /// </summary>
+        /// <param name="width">The width of the coordinate.</param>
```

Comment:
We don't return the coordinate, it is just size.

Reply:
Done